spec/factories/shopping_carts.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :shopping_cart do
```

Comment:
I don't have a problem with this. We will expand this factory as we add more attributes to ShoppingCart. Like associating it with a User, etc.